### DISCUSSION TITLE
PASSED MY AWS CLOUD PRACTITIONER EXAM CLF-02

### DISCUSSION THREAD

**Discussion by Senam-Kudjo** [kananinirav/AWS-Certified-Cloud-Practitioner-Notes#249] [General]
I just wanted to thank you @kananinirav for making this awesome repository. I just passed my CLF-02 hours ago. Every question I saw here somehow ended up in the exam as well.

**Comment by kananinirav** [comment]
@Senam-Kudjo Thanks you and Congratulations 🎉

**Comment by ballu24** [comment]
Congrats on passing the CLF-02. @Senam-Kudjo! Quick question for both. I will be taking the AWS Certified Cloud Practitioner Exam (CLF-C02) in 3 weeks. Are the practice exams associated to CLF - C02 or CLF - C01? @Senam-Kudjo @kananinirav, Any assistance is greatly appreciated.

**Reply by Senam-Kudjo** [reply]
CLF-C02

**Reply by Senam-Kudjo** [reply]
CLF-C02 ... I scored 951/1000 so just stick to this repo though some answers here a wrong so cross check with AI and official sources as well. Also learn about Cloud migration. I barely some cloud migration questions in this repo.

**Reply by ballu24** [reply]
Congrats! Thank you so much for the confirmation. I will be taking the exam in the next couple of weeks. Wil provide feedback! Hopefully I pass!

**Comment by mebaxo7568** [comment]
I am happy to share that I passed my certification CLF-C02 exam. Exam4Lead offered reliable study materials that closely matched the actual exam. https://www.exam4lead.com/amazon/clf-c02-dumps.html
